src/NzbDrone.Common/Http/HttpClient.cs

```diff
@@ -28,11 +28,14 @@ public class HttpClient : IHttpClient
         private readonly ICached<CookieContainer> _cookieContainerCache;
         private readonly ICached<bool> _curlTLSFallbackCache;
 
+        public bool forceCurl { get; set; }
```

Comment:
proper way to do this would be to separate out the dispatcher (Curl or WebClinet) into two different classes `IHttpDistapcher``CurlHttpDispatcher` and `ManagedHttpDispatcher`, then have a `DispatcherFactory` that you would take a `HttpReqeust` and it would return you an `IHttpDisptacher`.